docker_deploy/docker_manager_test.go

```diff
@@ -13,6 +13,21 @@ import (
 var imageName = "mcr.microsoft.com/dotnet/samples"
 var repoDigest = "sha256:d41fe80991d7c26ad43b052bb87c68a216a365c143623a62b5a5963fcdb77eb1"
 
+var options = map[string]interface{}{
+	"Hostname":   "my-container",
+	"User":       "root",
+	"Image":      "alpine",
+	"WorkingDir": "/app",
+	"StopSignal": "SIGTERM",
+}
+
+var hostOptions = map[string]interface{}{
+	"NetworkMode": "bridge",
+	"AutoRemove":  "true",
```

Comment:
[minor] if we are going to use a map[string]interface{} then true and false should be bools not strings, otherwise just create a map[string]string

Reply:
I'm actually not sure how this happened. We implement with bools not strings. Let me look at this closer.